recipe/recipe.yaml

```diff
@@ -7,29 +7,51 @@ package:
   version: ${{ version }}
 
 source:
-  url: https://github.com/MediaArea/MediaInfo/archive/refs/tags/v${{ version }}.tar.gz
-  sha256: "131b27960490887efee8db6a8471f98c37b478360060f9058d047e3c6a3afc44"
+  # repo contains symlinks which is hard to handle if we use tarball or zip.
+  - if: win
+    then:
+      path: ./build/
+    else:
+      url: https://github.com/MediaArea/MediaInfo/archive/refs/tags/v${{ version }}.tar.gz
+      sha256: "131b27960490887efee8db6a8471f98c37b478360060f9058d047e3c6a3afc44"
+      target_directory: src
 
 build:
-  skip: win
   script:
-    content: |
-      pushd Project/GNU/CLI
-      autoreconf -fi
-      ./configure --prefix="$PREFIX" || (cat config.log && false)
-      make install
-  number: 0
+    content:
+      - if: unix
+        then: |
+          pushd src/Project/GNU/CLI
+          autoreconf -fi
+          ./configure --prefix="$PREFIX" || (cat config.log && false)
+          make install
+
+        # repo contains symlinks which is hard to handle if we use tarball or zip.
```

Comment:
could you also add a reference to a rattler-build issue?

Reply:
there are multiple symlink issue, which should I ref?

https://github.com/prefix-dev/rattler-build/issues?q=is%3Aissue%20state%3Aopen%20symlink

Reply:
1671